store/populations.go

```diff
@@ -493,38 +508,32 @@ func getObsSeries(
 	if err != nil {
 		return nil, err
 	}
-	pbData := &pb.PopObsPlace{}
+	pbData := &pb.ChartStore{}
 	jsonpb.UnmarshalString(string(val), pbData)
-	ts := pb.ObsTimeSeries{PlaceName: pbData.Name, Data: map[string]float64{}}
-	for _, obs := range pbData.Observations {
-		if obs.MeasurementMethod != "DataCommonsAggregate" {
-			mmethod := strings.Replace(obs.MeasurementMethod, "dcAggregate/", "", -1)
-			if mmethod != statsVar.MeasurementMethod {
-				continue
+	switch x := pbData.Val.(type) {
+	case *pb.ChartStore_ObsTimeSeries:
+		result := &pb.ObsTimeSeries{}
+		result.Unit = x.ObsTimeSeries.GetUnit()
+		result.PlaceName = x.ObsTimeSeries.GetPlaceName()
+		result.IsDcAggregate = x.ObsTimeSeries.GetIsDcAggregate()
+		bestScore := lowestRank
+		for _, series := range x.ObsTimeSeries.SourceSeries {
+			key := rankKey{series.GetImportName(), series.GetMeasurementMethod()}
+			score, ok := statsRanking[key]
+			if !ok {
+				score = lowestRank
```

Comment:
nit: If not found, should we consider it having the highestRank, in that it will always be shown :)

Reply:
Hmm, never mind. This is the correct thing to do when you have new sources for the same statvar.